.github/workflows/tests_lgpumpi_cpp.yml

```diff
@@ -39,7 +39,7 @@ concurrency:
 
 jobs:
   cpp_tests:
-    if: ${{ contains(github.event.pull_request.labels.*.name, 'ci:use-multi-gpu-runner') && !contains(fromJSON('["schedule", "workflow_dispatch"]'), github.event_name) && github.event.pull_request.draft == false}}
+    if: ${{ github.event.pull_request.draft == false && (contains(github.event.pull_request.labels.*.name, 'ci:use-multi-gpu-runner') || contains(fromJSON('["schedule", "workflow_dispatch"]'), github.event_name)) }}
```

Comment:
# Question
Just to double check, does this have the correct behaviour when the event which triggered the workflow is `push` or `workflow_call`?

Reply:
Seems to work fine from manual check